### DISCUSSION TITLE
Possibilità di aggiungere nuovi voti ( nc, imp ...) e planning settimanale

### DISCUSSION THREAD

**Discussion by fxlduto** [iisgiua/giuaschool#715] [Idee e proposte]
Chiedo al gentilissimo staff di Giua di implementare per l'amministratore la funzione di modifica dei voti e del suo valore e colore... Chiedo inoltre di permettere ad ogni docente di caricare l'orario così da rendere la firma più veloce.

**Comment by trinko** [comment]
La segno come richiesta.
